### DISCUSSION TITLE
New Design: Analytics Tab for Asynchronous User Tests

### DISCUSSION THREAD

**Discussion by jvJUCA** [ruxailab/RUXAILAB#301] [Show and tell]
![Captura de ecrã de 2024-01-11 14-09-41](https://github.com/uramakilab/RUXAILAB/assets/128351990/20ea5dd4-b5b9-4946-bf7b-4173e3f99177)
Introducing our revamped Analytics tab for asynchronous user tests! Designed for enhanced data visualization and usability. We value your feedback to refine the design further.

Thanks!

**Comment by KarinePistili** [comment]
I liked the idea! It looks clean and useful. I'm just a little confused about the answers timeline chart, is it the quantity of answers in a period of time?

**Comment by gokul-123461** [comment]
[gsoc.2026 (1).pdf](https://github.com/user-attachments/files/26025778/gsoc.2026.1.pdf)
Hello,
I am Gokul Krishnan, a second-year B.Tech IT student. I am very interested in contributing to the RUXAILAB organization for GSoC 2026.
I have prepared a proposal for the project "Eye-Tracking Accuracy Benchmarking and Validation Tool", and I have attached it here. I would really appreciate it if you could take a look and share your feedback or suggestions for improvement.

Thank you for your time and guidance.
